biz.aQute.tester/testresources/aQute/tester/test/JUnit3NonStaticBC.java

```diff
@@ -0,0 +1,20 @@
+package aQute.tester.test;
```

Comment:
When you add a new folder for test resources to a project, you need to list that folder in 

`-builderignore: testresources`

So that the gradle build does not include that folder in dependency checking for the jar task.

Reply:
Fixed - I thought this would be the default though. Is there a good reason why this isn't the default?

Reply:
> Is there a good reason why this isn't the default?

Default for this project? I don't think it is a good standard in general. We should probably use normal maven/gradle source folder conventions `src/test/resources` instead of testresources and testdata...

Reply:
I mean generally. Perhaps by default the builder should ignore directories until it is explicitly told to use them in dependency considerations, rather than use them until it is explicitly told to ignore them. Agree about using the maven/gradle conventions.

Reply:
That will not work for many projects. For example biz.aQute.launcher/bnd contains information which is input to the bundle. So by default all folders not known to be output (e.g. bin, bin_test, generated) are assumed to contribute to the bundle unless you name them in -builderignore.